ratarmount.py

```diff
@@ -886,17 +897,17 @@ def _createIndex(
 
             # fmt: off
             fileInfo = (
-                ""                 ,  # 0 path
-                fname              ,  # 1
-                None               ,  # 2 header offset
-                0                  ,  # 3 data offset
-                fileSize           ,  # 4
-                tarInfo.st_mtime   ,  # 5
-                tarInfo.st_mode    ,  # 6
-                None               ,  # 7 TAR file type. Currently unused but overlaps with mode anyways
-                None               ,  # 8 linkname
-                tarInfo.st_uid     ,  # 9
-                tarInfo.st_gid     ,  # 10
+                ""                                          ,  # 0 path
+                fname                                       ,  # 1
+                None                                        ,  # 2 header offset
+                0                                           ,  # 3 data offset
+                fileSize                                    ,  # 4
+                tarInfo.st_mtime if tarInfo else 0          ,  # 5
+                tarInfo.st_mode if tarInfo else 0o444       ,  # 6
```

Comment:
Just a thought but I think it can be ignored because the permissions are not enforced anyway without the FUSE layer: If you have permission 0x444 and uid=0 and gid=0, then theoretically only root should be able to access those files.